db/install.xml

```diff
@@ -11,9 +11,9 @@
         <FIELD NAME="enabled" TYPE="int" LENGTH="4" NOTNULL="true" UNSIGNED="true" DEFAULT="1" SEQUENCE="false" PREVIOUS="course" NEXT="buttonstype"/>
         <FIELD NAME="buttonstype" TYPE="char" LENGTH="6" NOTNULL="true" DEFAULT="icon" SEQUENCE="false" COMMENT="Buttons should be text or icons" PREVIOUS="enabled" NEXT="backgroundcolour"/>
         <FIELD NAME="backgroundcolour" TYPE="char" LENGTH="20" NOTNULL="false" DEFAULT="#6666cc" SEQUENCE="false" PREVIOUS="buttonstype" NEXT="customusebackground"/>
-        <FIELD NAME="customusebackground" TYPE="int" LENGTH="4" NOTNULL="true" UNSIGNED="true" DEFAULT="0" SEQUENCE="false" PREVIOUS="backgroundcolour" NEXT="homebuttonshow"/>
-        <FIELD NAME="homebuttonshow" TYPE="int" LENGTH="4" NOTNULL="true" UNSIGNED="true" DEFAULT="1" SEQUENCE="false" PREVIOUS="customusebackground" NEXT="homebuttontype"/>
-        <FIELD NAME="homebuttontype" TYPE="int" LENGTH="4" NOTNULL="true" UNSIGNED="true" DEFAULT="1" SEQUENCE="false" PREVIOUS="homebuttonshow" NEXT="firstbuttonshow"/>
```

Comment:
I've not time to do a full review now - but why have you removed the homebuttontype field?